key-selector-component.js

```diff
@@ -26,14 +27,6 @@ function register(module) {
 /* @ngInject */
 function Ctrl($scope) {
   var self = this;
-
-  $scope.$watchCollection(function() {
-    return self.keys;
-  }, function() {
-    if(!self.selected || $.inArray(self.selected, self.keys) === -1) {
-      self.selected = self.keys[0] || null;
-    }
-  });
```

Comment:
I'd say we should keep this for backwards-compatibility, but I really was unhappy with it to begin with, so :heart: for getting rid of it.